scripts/create-pr.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
-PR_TITLE="chore (automated): OpenSauced updates"
 BRANCH_NAME="chore_automated_open-sauced-updates_$(date +%s)"
 
+if [[ -n "$CUSTOM_PR_TITLE" ]]; then
+  PR_TITLE="$CUSTOM_PR_TITLE"
+else
+  PR_TITLE="chore (automated): OpenSauced updates"
+fi
```

Comment:
If there is a default PR title as suggested above, this can be.

```suggestion
PR_TITLE="$CUSTOM_PR_TITLE"
```